config/config.go

```diff
@@ -144,20 +144,34 @@ type Gen2Config struct {
 
 // VPCProviderConfig configures a specific instance of a VPC provider (e.g. GT/GC/Z)
 type VPCProviderConfig struct {
-	Enabled              bool   `toml:"vpc_enabled" envconfig:"VPC_ENABLED"`
+	Enabled bool `toml:"vpc_enabled" envconfig:"VPC_ENABLED"`
+
+	//valid values (gc|g2), if unspecified, GC will take precedence(if both are specified)
+	//during e2e test, user can specify its own preferred type during execution
+	VPCTypeEnabled       string `toml:"vpc_type_enabled" envconfig:"VPC_TYPE_ENABLED"`
```

Comment:
`VPCTypeEnabled ` is not required as cli or api can point either ng or gc at a time not both.

Reply:
I would say e2e should also run independently for gc and ng

Reply:
We should expect the configuration can  contain all types of provider config at the same time and Enable FLAGs should be used to enable/disable.

Reply:
actually its not a separate provider for library which need different config